tools/autograd/derivatives.yaml

```diff
@@ -514,6 +514,14 @@
 - name: log2(Tensor self) -> Tensor
   self: grad / (self * 0.6931471805599453)
 
+- name: logaddexp(Tensor self, Tensor other) -> Tensor
```

Comment:
@albanD Take a look, would you?

Reply:
The formula looks good.
For testing, you want to add an entry [here](https://github.com/pytorch/pytorch/blob/378956b48143d5858cc183c885d39a7aa64182c9/torch/testing/_internal/common_methods_invocations.py#L563) to make sure the gradients will be properly checked.